webview/src/shared/components/tooltip/Tooltip.tsx

```diff
@@ -52,7 +52,8 @@ const TooltipRenderFunction: React.ForwardRefRenderFunction<
     isContextMenu = false,
     animation = false,
     followCursor: followCursorVal,
-    offset
+    offset,
+    maxWidth = 350
```

Comment:
The `maxWidth` of tooltips is 350 by default but we need our table tooltips to be a little wider than that so I added another Tippy prop.